app/models/addTimestampsToQuotes.rb

```diff
@@ -0,0 +1,3 @@
+class AddTimestampsToQuotes < ActiveRecord::Base
+
+end
```

Comment:
IS this really necessary? we could probably deal without this class right, it doesn't seem to do anything?